src/protocol.h

```diff
@@ -263,7 +263,7 @@ enum ServiceFlags : uint64_t {
     NODE_BLOOM = (1 << 2),
     // NODE_WITNESS indicates that a node can be asked for blocks and transactions including
     // witness data.
-    NODE_WITNESS = (1 << 3),
```

Comment:
When this activates, those bits will become dead code.

Consider the picture of the network six months after this activates, when SegWit2X (and therefore segwit) is active.

The long term view implies avoiding dragging early artifacts into long term code.